docs/about/index.rst

```diff
@@ -1,13 +1,37 @@
 .. _about:
 
-About
-=====
+About Fairlearn
+===============
```

Comment:
Should we add a short summary here for people who don't bother reading the mission statement?

Reply:
Interesting. I like the spirit of the idea, but I'm not sure I can write it even shorter :-) Any suggestions?

Reply:
Something like: "Fairlearn is a community-driven open source project on a mission to help data scientists assess and improve the fairness of AI systems."

Reply:
Although I guess "on a mission" sounds a bit like we're characters of some adventure tv show, so feel free to use a different phrase :')

Reply:
I have updated the first line to be the same as on our website.

Reply:
I personally would prefer to have both assessment in the main sentence as well, as this is an important part of the project and some people might be specifically looking for auditing tools. But it's not a hill I'm willing to die on so if people disagree we can leave it as is.

Reply:
Ahhh... shall we edit "improve" -> "assess and improve"? (And we should tweak likewise on our website.)

Reply:
Sounds good! 

Just as an FYI, this is what the website looks like on my phone:
![IMG_61D2AEE5A0BB-1](https://user-images.githubusercontent.com/24417440/119153923-f88c6580-ba51-11eb-8207-302eb6bf272a.jpeg)